packages/testing/test/database.test.ts

```diff
@@ -131,14 +131,14 @@ describe('Testing Module Tests', function() {
         }
       }`
     });
-    await expect(promise).to.be.rejectedWith(/UNAVAILABLE/);
+    await expect(promise).to.be.rejectedWith(/ECONNREFUSED/);
```

Comment:
Could you please fix this test case while we're at it? The test case name says `succeeds on valid input` but actually just asserts connection to be refused. I think we should introduce http mocks here (like using `nock` like other JS SDK packages) and assert it sends out the right request instead.

Reply:
Or maybe use emulators for this test?

Reply:
Good idea, moved tests to the emulators.